README.md

```diff
@@ -30,6 +30,10 @@ For usage examples check out the [full documentation](https://humeai.github.io/h
 - [Platform Documentation](https://help.hume.ai/basics/about-hume-ai)
 - [API Reference](https://docs.hume.ai)
 
+## Citation
+
+To cite Hume's models, datasets, APIs, or SDKs in your work please see [CITATIONS.md](CITATIONS.md).
```

Comment:
what about saying as well, 'and explore the scientific research that went into them, please'

Reply:
Good idea. I added a bit about the scientific research